### PR TITLE
Add ovs documentation

### PR DESCRIPTION
**Pull request by SchSeba**
Related to https://github.com/kubevirt/cluster-network-addons-operator/pull/183

This PR add documentation about the ovs cni plugin to the README file

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/cluster-network-addons-operator/196)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +184,20 @@ yum install NetworkManager
 systemctl start NetworkManager
 ```
 
+## Ovs
```
Comment (phoracek) [inline README.md]: Please make it Open vSwitch

File: README.md [hunk 2]
```diff
@@ -183,6 +184,20 @@ yum install NetworkManager
...
+
+The operator allows administrator to deploy [Ovs CNI plugin](https://github.com/kubevirt/ovs-cni/)
```
Comment (phoracek) [inline README.md]: OVS CNI

**Comment by SchSeba** [comment]
done

**Comment by phoracek** [comment]
LGTM, just README, no need for CI.

**Pull request merged**